src/redux/categories/categories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Categories = () => (
+  <h1>Under Construction</h1>
+);
+
+export default Categories;
```

Comment:
- [ ] There is a misunderstanding here. The categories.js file in redux is for setup functionality for redux. It is not for your components. Please keep this file empty for this milestone and create your components in your components folder. ♻️